src/main/java/org/folio/bulkops/service/LogFilesService.java

```diff
@@ -80,4 +84,14 @@ private void removeFiles(BulkOperation bulkOperation) {
       bulkOperation.setLinkToPreviewRecordsJsonFile(null);
     }
   }
+
+  public void deleteFileByOperationIdAndName(UUID operationId, String fileName) {
+    var baseName = FilenameUtils.getBaseName(fileName);
+    var csvPath = format(CSV_PATH_TEMPLATE, operationId, baseName);
+    var jsonPath = format(JSON_PATH_TEMPLATE, operationId, baseName);
+    remoteFileSystemClient.remove(csvPath);
```

Comment:
Just an idea - can we make these calls in parallel?

Reply:
Changed to remoteFileSystemClient.remove(String... paths)